handlers/dashboard.go

```diff
@@ -344,49 +349,89 @@ func DashboardDataProposals(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func DashboardWithdrawals(w http.ResponseWriter, r *http.Request) {
-
-	// [WIP]
+func DashboardDataWithdrawals(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", "application/json")
 
 	q := r.URL.Query()
 	validatorLimit := getUserPremium(r).MaxValidators
-	filterArr, err := parseValidatorsFromQueryString(q.Get("validators"), validatorLimit)
+	validators, err := parseValidatorsFromQueryString(q.Get("validators"), validatorLimit)
 	if err != nil {
 		http.Error(w, "Invalid query", 400)
 		return
 	}
-	filter := pq.Array(filterArr)
 
-	proposals := []struct {
-		Slot   uint64
-		Status uint64
-	}{}
+	draw, err := strconv.ParseUint(q.Get("draw"), 10, 64)
+	if err != nil {
+		logger.Errorf("error converting datatables data parameter from string to int: %v", err)
```

Comment:
I would suggest to always use utils.LogError / utils.LogFatal, which will do all that for you.

Reply:
replaced with utils.LogError